cluster/gce/config-default.sh

```diff
@@ -95,6 +95,11 @@ if [[ "${ENABLE_NODE_AUTOSCALER}" == "true" ]]; then
   TARGET_NODE_UTILIZATION="${KUBE_TARGET_NODE_UTILIZATION:-0.7}"
 fi
 
+# Optional: Create autoscaler for cluster's pods.
+# Experimental feature, not ready for production use.
+ENABLE_HORIZONTAL_POD_AUTOSCALER="${KUBE_ENABLE_HORIZONTAL_POD_AUTOSCALER:-false}"
+ENABLE_EXPERIMENTAL_API="${ENABLE_HORIZONTAL_POD_AUTOSCALER}"
```

Comment:
Shouldn't this only enable and never disable experimental API? I imagine someone might want to enable it by hand, without autoscaler, right? 

You should also use somehow the default value (similarly to other variables)